File: app/src/main/java/tech/destinum/recorderis/pojo/Document.java
package tech.destinum.recorderis.pojo;

public class Document {
    private String mName, symbol;
    private int id;

    public Document(String name, String symbol, int id) {
        mName = name;
        this.symbol = symbol;
        this.id = id;
    }

    public String getName() {
        return mName;
    }

    public void setName(String name) {
        mName = name;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }


    public String getSymbol() {
        return symbol;
    }

    public void setSymbol(String symbol) {
        this.symbol = symbol;
    }
}

File: app/src/main/java/tech/destinum/recorderis/Data/Entities/Document.kt
package tech.destinum.recorderis.Data.Entities

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "documents")
data class Document(@ColumnInfo(name = "doc_name") var name: String,
               @ColumnInfo(name = "doc_symbol") var symbol: String){
    @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "document_id")
    var id: Int? = null
}
